components/VoteHistoryTable/VoteHistoryTable.tsx

```diff
@@ -0,0 +1,60 @@
+import { usePanelContext } from "hooks";
+import styled from "styled-components";
+import { VoteT } from "types";
+import { VoteHistoryTableRow } from "./VoteHistoryTableRow";
+
+interface Props {
+  votes: VoteT[];
+}
+export function VoteHistoryTable({ votes }: Props) {
+  const { openPanel } = usePanelContext();
+  const headings = ["Vote", "Staking", "Voted", "Correctness", "Score"];
+
+  return (
+    <Table>
+      <Thead>
+        <TheadTr>
+          {headings.map((heading) => (
+            <Th scope="col" key={heading}>
+              {heading}
+            </Th>
+          ))}
+        </TheadTr>
+      </Thead>
+      <Tbody>
+        {votes.map((vote) => (
+          <VoteHistoryTableRow
+            vote={vote}
+            key={vote.uniqueKey}
+            onVoteClicked={() => {
+              openPanel("vote", vote);
```

Comment:
clicking on a vote number opens up a new panel (vote panel) for that vote. this new panel covers the existing (history) panel. when the user clicks the close button on the vote panel, that panel is removed, taking the user back to the history panel.